fem/complex_fem.hpp

```diff
@@ -444,11 +496,28 @@ class ParSesquilinearForm
    ParBilinearForm *pblfr;
    ParBilinearForm *pblfi;
 
+   /* These methods check if the real/imag parts of the
+   sesqulinear form are not empty */
+   bool RealInteg();
+   bool ImagInteg();
+
 public:
    ParSesquilinearForm(ParFiniteElementSpace *pf,
                        ComplexOperator::Convention
                        convention = ComplexOperator::HERMITIAN);
 
+   /** @brief Create a ParSesquilinearForm on the ParFiniteElementSpace @a pf, using the
+       same integrators as the ParBilinearForms @a pbfr and @a pbfi .
+
+       The pointer @a pf is not owned by the newly constructed object.
+
+       The integrators are copied as pointers and they are not owned by
+       the newly constructed ParSesquilinearBilinearForm. */
```

Comment:
ParSesquilinearBilinearForm should be ParSesquilinearForm.

Reply:
Done!